.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: go
 
 go:
- - 1.11.4
+ - 1.12
```

Comment:
Go 1.11.4 was used in the tidepool repository. Why do we upgrade to Go 1.12 in this PR ?

Reply:
Projects using go-common was updated to 1.12. It make no sens of having a dep using an older version. It will be compiled by go 1.12 in the final binaries anyway.

Reply:
Not all projects were updated, hydrophone is still in Go 1.11 for example.

Reply:
Yes but hydrophone was not yet updated for this new go-common version. It still use the previous release.

Reply:
Hydrophone uses GoDep for its packages dependecies. And references the master branch not a tag (because of GoDep that must point to a branch...).
 
https://github.com/mdblp/hydrophone/blob/dblp/Gopkg.toml

Reply:
I don't think it is a problem, here my arguments:
- Gopkg.lock has a reference on the commit tag https://github.com/mdblp/hydrophone/blob/9ec8548ca515c7986d889842048905d2fa8d0b04/Gopkg.lock#L102
- The sources are fetch then compiled, so it will be compiled with go 1.11 anyway.
- New packages use go 1.12, so if kept in 1.11 you may say: new version don't use the same version of go lang.
- The next version of Tidepool go-common will use go 1.12 with no special change (https://github.com/tidepool-org/go-common/commit/cda61c2c2308e60acba70d3aa7413b86735f7f19)
- We already know it compile & run with go 1.11, because it is the version used by Tidepool. But we want to know if there is no problem in 1.12 for some dependencies.